docs/General/buildRPC.md

```diff
@@ -7,15 +7,16 @@ The Remote Provisioning Client (RPC) communicates with the Manageability Engine
 
 [![RPC](../assets/images/RPC_Overview.png)](../assets/images/RPC_Overview.png)
 
-### Build RPC
+## Build RPC
 
-We leverage GitHub Actions as a means to build RPC automatically leveraging Github's CI/CD Infrastructure. This avoids having to deal with the challenges of getting your build environment just right on your local machine and allows you to get up and running much faster. However, if you wish to do this locally, please follow the instructions [here](../Microservices/RPC/buildRPC_Manual.md).
+We leverage GitHub Actions as a means to build RPC automatically leveraging Github's CI/CD Infrastructure. This avoids having to deal with the challenges of getting your build environment just right on your local machine and allows you to get up and running much faster. However, if you wish to do this locally, please follow the instructions [here](../Microservices/RPC/buildRPC_Manual.md). Additionally, we support Docker for RPC as well, skip to [here](#docker-build) for the command to build with Docker.
```

Comment:
Ya know, for these redirections, I might say stuff this way instead:
"Additionally, we support Docker for RPC as well, skip to the section [Docker Build](#docker-build) to see the command to see the Docker build command."

It's less scary if you have some idea of where a thing is going to take you. 

I feel like "here" is maybe something we should think about removing. I'm guilty of it too, and it comes into play when I feel this need to shorten everything. Like "To yadah yadah, go [here](xxx)." But the reason that I'm getting that feeling is that there is probably too much verbosity before the here statement. And I'm focusing on the editing the wrong thing.

Reply:
Sorry, I'm trying to multi-task. What I meant to type was this: 

We leverage GitHub Actions using Github's CI/CD Infrastructure to build RPC automatically. This eliminates the challenges of configuring your local build environment and promotes a quicker path to installing and using the software. However, if you wish to build locally, please follow the instructions in [Build & Run RPC (Manual)](../Microservices/RPC/buildRPC_Manual.md). Additionally, to build the RPC with Docker commands, skip to [Docker Build](#docker-build). 

It's a bit shorter, maybe more precise. The last sentence of the paragraph in the checked-in version was a run on sentence. So, it fixes that too.